repose-aggregator/src/docs/resources/images/performance-test-framework.gliffy

```diff
@@ -0,0 +1 @@
.basic_v1.default.line","order":20015,"lockAspectRatio":false,"lockShape":false,"constraints":{"constraints":[],"startConstraint":{"type":"StartPositionConstraint","StartPositionConstraint":{"nodeId":81,"py":0.29289321881345237,"px":1.0}},"endConstraint":{"type":"EndPositionConstraint","EndPositionConstraint":{"nodeId":124,"py":0.5,"px":0.0}}},"graphic":{"type":"Line","Line":{"strokeWidth":2.0,"strokeColor":"#980000","fillColor":"none","dashStyle":null,"startArrow":0,"endArrow":2,"startArrowRotation":"auto","endArrowRotation":"auto","interpolationType":"linear","cornerRadius":null,"controlPath":[[-7.0,3.3354124636404663],[84.5,3.3354124636404663],[84.5,-88.63157894736844],[146.0,-88.63157894736844]],"lockSegments":{"1":true},"ortho":true}},"linkMap":[],"children":[],"hidden":false,"layerId":"bGfIZXoA21tW"},{"x":517.0,"y":441.0,"rotation":0.0,"id":161,"width":68.0,"height":45.0,"uid":"com.gliffy.shape.basic.basic_v1.default.line","order":20016,"lockAspectRatio":false,"lockShape":false,"constraints":{"constraints":[],"startConstraint":{"type":"StartPositionConstraint","StartPositionConstraint":{"nodeId":81,"py":0.29289321881345237,"px":1.0}},"endConstraint":{"type":"EndPositionConstraint","EndPositionConstraint":{"nodeId":128,"py":0.2928932188134525,"px":1.1102230246251563E-16}}},"graphic":{"type":"Line","Line":{"strokeWidth":2.0,"strokeColor":"#980000","fillColor":"none","dashStyle":null,"startArrow":0,"endArrow":2,"startArrowRotation":"auto","endArrowRotation":"auto","interpolationType":"linear","cornerRadius":null,"controlPath":[[-10.0,-1.6645875363595337],[31.5,-1.6645875363595337],[31.5,43.58921363323111],[73.0,43.58921363323111]],"lockSegments":{},"ortho":true}},"linkMap":[],"children":[],"hidden":false,"layerId":"bGfIZXoA21tW"}],"layers":[{"guid":"bGfIZXoA21tW","order":2,"name":"Interactions","active":false,"locked":false,"visible":true,"nodeIndex":18},{"guid":"8pbi2R4AZnwP","order":1,"name":"Hosts","active":true,"locked":false,"visible":true,"nodeIndex":94},{"guid":"TIAblKBtzODd","order":0,"name":"Base","active":false,"locked":false,"visible":true,"nodeIndex":26}],"shapeStyles":{},"lineStyles":{"global":{"fill":"none","stroke":"#980000","strokeWidth":2,"endArrow":2,"orthoMode":0,"startArrow":0}},"textStyles":{"global":{"face":"Arial","size":"12px","color":"#000000"}}},"metadata":{"title":"untitled","revision":0,"exportBorder":false,"loadPosition":"default","libraries":["com.gliffy.libraries.mru","com.gliffy.libraries.basic.basic_v1.default","com.gliffy.libraries.flowchart.flowchart_v1.default","com.gliffy.libraries.swimlanes.swimlanes_v1.default","com.gliffy.libraries.uml.uml_v2.class","com.gliffy.libraries.uml.uml_v2.sequence","com.gliffy.libraries.uml.uml_v2.activity","com.gliffy.libraries.erd.erd_v1.default","com.gliffy.libraries.ui.ui_v3.containers_content","com.gliffy.libraries.ui.ui_v3.forms_controls","com.gliffy.libraries.images"],"autosaveDisabled":false,"lastSerialized":1493933904951},"embeddedResources":{"index":0,"resources":[]}}
```

Comment:
This looks like straight json. Can gliffy still load it if you format it? I ask because I'd hope that they don't shift things around in the output too much, and it'd be much easier to see changes made later if that's true.